CONTRIBUTING.md

```diff
@@ -463,9 +463,9 @@ The initial values of the _.env_ will be copied from the _.env.example_ file. Ho
 [PostgreSQL](https://www.postgresql.org/download) is our database and [TypeORM](https://typeorm.io/) is used to map tables to JS objects.
 
 ### Schema
-Our [database schema](https://freecodecamp.github.io/chapter/) and [ER Diagram](https://freecodecamp.github.io/chapter/relationships.html) are hosted online on a GitHub pages domain.
+Our [database schema and ER Diagrams are available online](https://opensource.freecodecamp.org/chapter/) using a custom GitHub pages domain using the [SchemaSpy](http://schemaspy.org/) format.
 
-This is [currently manually generated and updated](https://github.com/freeCodeCamp/chapter/issues/54#issuecomment-799653569) on the _gh-pages_ branch by running [SchemaSpy](http://schemaspy.org/).
+Updates to the _gh-pages_ branch and [online schema](https://opensource.freecodecamp.org/chapter/) are automatically triggered by commits to the _main_ branch, so we no longer require [manual steps](https://github.com/freeCodeCamp/chapter/issues/54#issuecomment-799653569).
```

Comment:
```suggestion
Updates to the _gh-pages_ branch and [online schema](https://opensource.freecodecamp.org/chapter/) are automatically triggered by commits to the _main_ branch.
```
Do we need to keep a link to the manual approach?

Reply:
I left in in there in case someone wanted to run it locally for some reason. 

I could link the works "automatically triggered" to the PR that you did and put a note back to the manual method in that PR to keep a breadcrumb back to those if people wanted more info about how we were automatically and manually making it work.

Reply:
Got you, that sounds good.